ideascube/monitoring/tests/test_migration_unique_serial.py

```diff
@@ -0,0 +1,67 @@
+from ideascube.tests.helpers import migration_test
+
+
+@migration_test(
+    migrate_from=[('monitoring', '0005_auto_20161027_0801')],
+    migrate_to=[('monitoring', '0006_unique_serial')]
+)
+def test_do_not_break_already_unique_serial(migration):
+    Specimen = migration.old_apps.get_model('monitoring', 'Specimen')
+    StockItem = migration.old_apps.get_model('monitoring', 'StockItem')
+
+    stockItem = StockItem()
```

Comment:
Same questions about saving the `stockItem` and `specimen` as in the other function.